CONTRIBUTING.md

```diff
@@ -32,16 +32,14 @@ If you're worried or don’t know where to start, check out our next section exp
 -   Alternatively you can use your machine's LAN IP.
 
 7. Deploy the database with `npm run prisma:migrate`
-8. Start the backend with `npm run start`
-9. Go to the `web/crux-ui` directory: `cd web/crux-ui`
-10. Install dependencies `npm ci`
-11. Copy the _env.example_ file as _.env_ `cp .env.example .env`
-12. Start the frontend with `npm run start`
-13. After you navigated to `localhost:8000` (this is the default Traefik port) you will see a Login screen
-14. Register an account with whatever e-mail address you see fit (doesn't have to be valid one)
-15. Navigate to `localhost:4436` where you will find your mail as all outgoing e-mails will land here
-16. Open your e-mail message and using the link inside you can activate your account
-17. Fruitful contributing! 🎬
+8. Start the backend in production mode with `npm run start`
```

Comment:
There seems to be some confusion here. This is the development section, so we'd like to start the backend in development mode. The command is OK though.

Reply:
There is a command alias for start: `npm start`, run could be omitted.